modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/AuditGraphApp.js

```diff
@@ -62,9 +62,11 @@ const data = [
 	},
 ];
 
+const langDir = 'ltr';
```

Comment:
I presume this is only temporarily hard-coded, right?

Reply:
Yes!